spec/turnsole/heliotrope/service_spec.rb

```diff
@@ -3,23 +3,27 @@
 RSpec.describe Turnsole::Heliotrope::Service do
   service = described_class.new
 
+  #
+  # Cannot create components because noid has to be valid!!!
```

Comment:
@gkostin1966 Is this right? Did these tests ever work?

Reply:
They do, but you need to have a running heliotrope instance in development to run the tests. It literally will run the turnsole API specs against a separate running heliotrope instance, adding and removing records. From what I can tell that how this has always worked...

Reply:
Yes, the test have always run against an instance of heliotrope.  It was either that or mock the heliotrope API which is probably more work than it is worth and could possibly lead to false positives if the mock is out of sync with the actual heliotrope API.

Reply:
Also, in the beginning the noid didn't have to exist for a component to be created.  Later something changed which require the noid to exist when creating a component.  If I'm not mistaken it has to do with entity validation but I am not certain.  Anyway the test was commented out as an artifact to this change in requirements.